libnetwork/sandbox_unsupported.go

```diff
@@ -30,3 +30,9 @@ func (sb *Sandbox) populateNetworkResources(*Endpoint) error {
 	// not implemented on Windows (Sandbox.osSbox is always nil)
 	return nil
 }
+
+func (sb *Sandbox) IPv6Enabled() (enabled, ok bool) {
+	// Not implemented on Windows (Sandbox.osSbox is always nil).
+	// Assume the container supports IPv6, in case there's a Windows IPAM driver that does.
+	return true, true
```

Comment:
Nope. https://github.com/moby/moby/blob/1f539a6e8581ae33f19d7e654404f8f0f74e36e1/libnetwork/drivers/windows/windows.go#L240-L242

```suggestion
	// None of the Windows container network drivers currently support IPv6.
	return false, true
```

Reply:
Ok.